calendar/page.js

```diff
@@ -532,6 +574,14 @@ function buildCalendarEventObject(record, colTypes) {
     isAllday,
     category: 'time',
     state: 'Free',
+    raw,
+    ...raw,
```

Comment:
add explanation comments for this two lines please, because why they are here is not clear.